modules/radsBidAdapter.js

```diff
@@ -31,33 +30,35 @@ export const spec = {
       let endpoint = isDev ? ENDPOINT_URL_DEV : ENDPOINT_URL;
 
       let payload = {};
-      if (isVideoRequest(bidRequest)) {
-        let vastFormat = params.vastFormat || DEFAULT_VAST_FORMAT;
+      if (isBannerRequest(bidRequest)) {
+        let size = getBannerSizes(bidRequest)[0];
```

Comment:
If you only plan on supporting one size you should update your docs to mention this, many publishers send multiple sizes in their requests.